os-eap-hawkular/added/standalone.conf

```diff
@@ -1,5 +1,10 @@
 # add hawkular options
 if [ -n "${AB_HAWKULAR_REST_URL:+x}" ]; then
+  if [ -f $JBOSS_HOME/bin/launch/logging.sh ]; then
+    source $JBOSS_HOME/bin/launch/logging.sh
+  fi
+  # CLOUD-3412 - if hawkular is enabled the note that it is deprecated and will be removed in a future release
+  log_warning "Please note that the Hawkular Java Agent has been deprecated and will be removed in a future release."
```

Comment:
Should this be in the if block above since log_warning would fail if logging.sh wasn't sourced?

Reply:
The conditional sourcing is really only so we can have the file not be present and redefine log_warning (like in bats tests etc). But yeah, this is unnecessarily complicating it, I'll simplify it.

Reply:
I've updated this to work the same way that we used for 7.2-openjdk11 https://github.com/jboss-container-images/jboss-eap-modules/blob/7.2.x-openjdk11/os-eap-hawkular/added/standalone.conf#L1

Logging is required in both of these images due to the dependency on java s2i, here: https://github.com/jboss-openshift/cct_module/blob/0.28.1/jboss/container/java/s2i/bash/module.yaml#L22